changelogs/fragments/50322-openssl-path-error.yml

```diff
@@ -0,0 +1,7 @@
+bugfixes:
+- "openssl_certificate - fix error when ``path`` contains a file without path."
```

Comment:
How about:

`openssl_* - fix error when ``path`` contains a relative path.`

On this and the following lines?

Reply:
Good idea. I initially thought I'd prefer the explicit list so that people grepping for their used modules will find it, but this is a really minor thing where it shouldn't be a problem if people don't notice this change. I've updated the changelog.

Reply:
Oh, I rather was talking about the `contains a file without path` vs. `contains a relative path` part. ;-)

Reply:
Alternatively maybe `contains a file name without path`?

Reply:
Ah, sorry. I guess it was too late for me ;-) Anyway, `file name without path` is better, since relative paths are not a problem. The problem is if there is no path at all, neither absolute nor relative.